streams/src/main/java/org/apache/kafka/streams/internals/StreamsConfigUtils.java

```diff
@@ -42,12 +42,12 @@ public enum ProcessingMode {
             this.name = name;
         }
     }
-    
-    @SuppressWarnings("deprecation")
+
+    // TODO cleanup
```

Comment:
Will be addressed in a follow up PR -- for now, just replacing removed variables with hard-coded strings below.